### ISSUE TITLE
Add Idioms to the sidebar menu of the Jewish Neo-Aramaic Living Dictionary

### ISSUE THREAD

**Issue by Danble** [livingtongues/living-dictionaries#173]
The Jewish Neo-Aramaic community requested us that they need to add idioms for their dictionary. It would be a component almost identical to what we're using in Grammar:
![image](https://user-images.githubusercontent.com/43384963/170386644-d082964f-109b-4814-aaca-a9b643883ca8.png)

**Comment by AnnaLuisaD** [comment]
only managers and collaborators can edit the Idioms tab, but all people browsing should be able to see it (and not edit)

**Comment by jacob-8** [comment]
Idioms should be dictionary entries just like everything else. They are of the phrase sort, like "hit the nail on the head" - not single words usually and not full sentences. What's needed here is our previously talked about feature of custom tags. That dictionary can create an "idiom" tag and then people can just filter by that tag when they want to see all the idioms. However, if others also want to add idioms, then another idea is perhaps "idiom" should be listed as a part of speech.

If we create a separate page for idioms, basically just a rich-text box like grammar and about, then they no longer have access to any of the entry features: audio, video, images, search, etc... - in my opinion this would be a step backwards.

**Comment by AnnaLuisaD** [comment]
yes, I told them they should also be adding these idioms as separate entries so they can actually be searched in the database. but they did request an extra tab just so someone could read all Idioms at once. I guess we could deny their request once we talk about it in the group meeting

**Comment by AnnaLuisaD** [comment]
Greg says that the Idioms tab could be useful to this particular team, but not particular useful to the rest of the platform. Obviously we would not deploy it site-wide. maybe we should just have a new semantic domain "Idiom" tag that people can use to attach to their entries. the question if we want to include this item as a semantic domain tag or as a part of speech tag. 

Like the example "hit the nail on the head" is a verb phrase, which we could label as such in the parts of speech. but we need to reflect on how this should also be tagged as an Idiom. 

Greg is reflecting on how we will tag it and we can come back to that in the next group meeting. we are in agreement that we need an "Idiom" tag.
